src/models/attachment.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * File or JSON attachments
+ * @example [{
+ *   "id": "string",
+ *   "filename": "string",
+ *   "binary_blob": {
+ *     "size": 0,
+ *     "type": "string"
+ *   },
+ *   "datetime": "2023-03-16T13:18:42.357Z"
+ * }]
+ */
+export interface Attachment {
+  /**
+   * uuid generated using knex
+   */
+  id?: string
```

Comment:
use `UUID` type?

Reply:
created before you created and it has been updated.